src/file/UserToFileWrite.java

```diff
@@ -0,0 +1,57 @@
+package file;
+
+import converter.UserToStringConverter;
+import exception.FileWriteException;
+import model.User;
+
+import java.io.IOException;
+import java.util.List;
+
+public class UserToFileWrite {
+
+    private static final String FILE_NAME = "Information_about_users";
+    private final UserToStringConverter userToStringConverter;
+    private final java.io.FileWriter writer;
+    private final String LINE_WRAPPING = "\n";
```

Comment:
1. Constants are also `private static final`
2. Let's name it like `LINE_SEPARATOR`